### ISSUE TITLE
Multi-Stage Dockerfile without latest tag triggers policy CKV_DOCKER_7

### ISSUE THREAD

**Issue by MMerzinger** [bridgecrewio/checkov#1450]
**Describe the bug**
Scanning a multi stage Dockerfile without any stage relying on a latest tag results in the violation of policy CKV_DOCKER_7 "Ensure the base image uses a non latest version tag".

**To Reproduce**
1. Create a Dockerfile (I generated one via VS Code on a fresh dotnet webapi project)
```Dockerfile
FROM mcr.microsoft.com/dotnet/aspnet:3.1-alpine AS base
WORKDIR /app
EXPOSE 5000

ENV ASPNETCORE_URLS=http://+:5000

# Creates a non-root user with an explicit UID and adds permission to access the /app folder
# For more info, please refer to https://aka.ms/vscode-docker-dotnet-configure-containers
RUN adduser -u 5678 --disabled-password --gecos "" appuser && chown -R appuser /app
USER appuser

FROM mcr.microsoft.com/dotnet/sdk:3.1-focal AS build
WORKDIR /src
COPY ["code/app/demo/demo.csproj", "code/app/demo/"]
RUN dotnet restore "code/app/demo/demo.csproj"
COPY . .
WORKDIR "/src/code/app/demo"
RUN dotnet build "demo.csproj" -c Release -o /app/build

FROM build AS publish
RUN dotnet publish "demo.csproj" -c Release -o /app/publish

FROM base AS final
WORKDIR /app
COPY --from=publish /app/publish .
ENTRYPOINT ["dotnet", "demo.dll"]
```
2. Run a Checkov scan
```bash
docker run -it --rm -v $(pwd)/code/app/demo/:/app bridgecrew/checkov --framework dockerfile -f /app/Dockerfile

       _               _              
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V / 
  \___|_| |_|\___|\___|_|\_\___/ \_/  
                                      
By bridgecrew.io | version: 2.0.318 

dockerfile scan results:

Passed checks: 4, Failed checks: 2, Skipped checks: 0

Check: CKV_DOCKER_1: "Ensure port 22 is not exposed"
        PASSED for resource: /app/Dockerfile.
        File: /app/Dockerfile:1-1
        Guide: https://docs.bridgecrew.io/docs/ensure-port-22-is-not-exposed

Check: CKV_DOCKER_5: "Ensure update instructions are not use alone in the Dockerfile"
        PASSED for resource: /app/Dockerfile.
        File: /app/Dockerfile:1-1
        Guide: https://docs.bridgecrew.io/docs/ensure-update-instructions-are-not-used-alone-in-the-dockerfile

Check: CKV_DOCKER_8: "Ensure the last USER is not root"
        PASSED for resource: /app/Dockerfile.USER
        File: /app/Dockerfile:9-9

Check: CKV_DOCKER_3: "Ensure that a user for the container has been created"
        PASSED for resource: /app/Dockerfile.USER
        File: /app/Dockerfile:9-9
        Guide: https://docs.bridgecrew.io/docs/ensure-that-a-user-for-the-container-has-been-created

Check: CKV_DOCKER_7: "Ensure the base image uses a non latest version tag"
        FAILED for resource: /app/Dockerfile.FROM
        File: /app/Dockerfile:19-19

                19 | 


Check: CKV_DOCKER_2: "Ensure that HEALTHCHECK instructions have been added to container images "
        FAILED for resource: /app/Dockerfile.
        File: /app/Dockerfile:1-1
        Guide: https://docs.bridgecrew.io/docs/ensure-that-healthcheck-instructions-have-been-added-to-container-images

                1 | FROM mcr.microsoft.com/dotnet/aspnet:3.1-alpine AS base
```
3. As you can see - policy check "CKV_DOCKER_7" failed, although no latest tag inside the Dockerfile.

**Expected behavior**
My expectation is that the policy check "CKV_DOCKER_7" is successful as the Dockerfile does not use any latest tag.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Checkov Version 
 ```bash
docker run -it --rm bridgecrew/checkov --version
2.0.318
```

**Comment by tophersmith** [comment]
https://github.com/bridgecrewio/checkov/pull/1899

**Comment by gruebel** [comment]
This should be fixed now in version 2.0.549. @tophersmith thanks for the fix 🚀
